submissions/ylizma/Main.java

```diff
@@ -53,6 +55,7 @@ private static void writeResultToFile(Map<String, Double> cityPrices, Map<String
                                     });
                         }
                 );
+        result.setLength(result.length() - 1);
```

Comment:
why is that ?

Reply:
i removed the last break line, so i have only the outpout data.